tools/pipelines/templates/include-publish-npm-package-steps.yml

```diff
@@ -108,23 +108,23 @@ steps:
                   exit 1
                 fi
             else
-              feedId=$(grep -oP '_packaging/\K[^/]+(?=/npm/)' publish_log)
+              feedName=$(grep -oP '_packaging/\K[^/]+(?=/npm/)' publish_log)
 
-              if [[ $release == "release" && $feedId == "build" ]]; then
+              if [[ "$(release)" == "release" && $feedName == "build" ]]; then
                 packageVersion=$(grep -oP 'npm notice version:\s+\K[^\s]+' publish_log)
 
                 if [ "${{ parameters.artifactPath }}" == "scoped" ]; then
-                  scopes=("fluidframework" "fluid-tools" "fluid-internal" "fluid-experimental")
+                  scopes=("fluidframework" "fluid-tools" "fluid-internal" "fluid-experimental" "fluid-private" "fluid-example")
                   for scope in "${scopes[@]}"; do
                       if [[ "$packageName" == "$scope"* ]]; then
                           packageName="@${packageName/$scope-/$scope/}"
                       fi
                   done
                 fi
 
-                baseURL="${${{ parameters.feedUrl }}%/registry/}"
-                feedPromotionUrl="${baseURL/_packaging/_apis/packaging/feeds}"
+                feedPromotionUrl="$(ado-feeds-api-base)/packaging/feeds/${feedName}/npm/${packageName}/versions/${packageVersion}?api-version=7.1-preview.1"
```

Comment:
The logic here might actually have worked out before, but IMO it was a bit hard to follow and it was easier to construct the REST API we needed to call directly rather than manipulate the registry URL.